FirebaseMLModelDownloader/proto/firebase_ml_log_sdk.pb.swift

```diff
@@ -1,677 +0,0 @@
-// DO NOT EDIT.
```

Comment:
Could you please remind why you're deleting the protobuf files?

Reply:
I already have the same files under `Sources/proto` so these are duplicate. Apologies for the confusion!